Content.Shared/_Umbra/Preferences/Loadouts/Effects/PersonalItemLoadoutEffect.cs

```diff
@@ -0,0 +1,49 @@
+using System.Diagnostics.CodeAnalysis;
+using Robust.Shared.Player;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Preferences.Loadouts.Effects;
```

Comment:
```suggestion
namespace Content.Shared._Umbra.Preferences.Loadouts.Effects;
```
Namespace should match the directory structure.